src/Veldrid/OpenGL/OpenGLCommandExecutor.cs

```diff
@@ -1715,6 +1729,33 @@ public void CopyBuffer(DeviceBuffer source, uint sourceOffset, DeviceBuffer dest
             _stagingMemoryPool.Free(block);
         }
 
+        private void CopySwapchainFramebuffer(
+            OpenGLTexture dst,
+            uint srcX, uint srcY,
+            uint dstX, uint dstY, uint dstZ, uint dstMipLevel, uint dstLayer,
+            uint width, uint height)
+        {
+            glBindFramebuffer(FramebufferTarget.ReadFramebuffer, 0);
```

Comment:
This should try using `SetSwapchainFramebuffer` first for platforms which have a swapchain framebuffer that's not zero (i.e. iOS).